conftest.py

```diff
@@ -111,14 +111,15 @@ def device_params(request):
 @pytest.fixture(scope="function")
 def device(request, device_params):
     import tt_lib as ttl
+    import ttnn
 
     device_id = request.config.getoption("device_id")
     request.node.pci_ids = [ttl.device.GetPCIeDeviceID(device_id)]
```

Comment:
Is there still not a function `ttnn` equivalent that does this?

Reply:
Hi @tt-rkim . It is handled in separate PR. Thankyou.